.circleci/config.yml

```diff
@@ -89,6 +98,7 @@ jobs:
       - checkout
       - run: sudo apt-get install -y sqlite3 libqt5x11extras5
       - *run_tests
+      - *run_lint
```

Comment:
we can always switch this later to run linters before tests if it causes headache. you've already mentioned the tradeoff here, so let's see how it goes as is!